tools/checked-c-convert/ConstraintBuilder.cpp

```diff
@@ -12,6 +12,12 @@
 using namespace llvm;
 using namespace clang;
 
+
+// flags
+// constraint all the arguments to a function
```

Comment:
Typo: this should be: "constrain all the arguments to a function".

Reply:
Fixed.